specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2022-10-01-preview/settings.json

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
As shown in the Breaking Changes report in your swagger PR, adding new paths/operations in an existing preview/stable version is a breaking change. So, you need to add new paths/operations in a new preview version.

In case, you have justification to include new paths in an existing API version, reach out to Azure Breaking Changes Reviewers <azbreakchangereview@microsoft.com> for exception and approval, and then, ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.

Reply:
This is existing preview api, we are isolating it into different file. This version was never published even it was added into the swagger.